microcontroller/src/error_handler.c

```diff
@@ -9,6 +15,10 @@ void _Error_Handler(char *file, int line) {
 
 void assert_failed(uint8_t* file, uint32_t line) {
 
+  // Unused parameters for now
```

Comment:
Shouldnt there be something useful here?
I mean to let us know that it's broken?

Reply:
We can add it later (AMPEROSE_DEBUG or something)